include/engine/datafacade/shared_datafacade.hpp

```diff
@@ -276,11 +276,6 @@ class SharedDataFacade final : public BaseDataFacade
 
     void LoadCoreInformation()
     {
-        if (data_layout->num_entries[storage::SharedDataLayout::CORE_MARKER] <= 0)
-        {
-            return;
-        }
-
```

Comment:
Is this general clean-up? Was that related to the bug? I am missing the context here on why this was here in the first place.